packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -6,16 +6,16 @@
 
 %{!?python_sitelib: %global python_sitelib %(%{__python} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
 
-%global build_agent (0%{?suse_version} == 0)
+%global build_agent (0%{?suse_version} == 0) && ((0%{?fedora} > 28) || (0%{?rhel} > 0))
```

Comment:
This update adds support for newer dnf versions which isn't compatible with old, we made the decision to just drop agent support for fedora 27 and 28 as part of this.